modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/shibari.dm

```diff
@@ -0,0 +1,230 @@
+/obj/item/stack/shibari_rope
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_items/lewd_items.dmi'
+	icon_state = "shibari_rope"
+	name = "Shibari ropes"
+	desc = "Coil of bondage ropes."
+	amount = 1
+	merge_type = /obj/item/stack/shibari_rope
+	singular_name = "ropes"
+
+	///color
+	var/current_color = "pink"
+	///if the thing's color has been changed or not
+	var/color_changed = FALSE
+	///cache for radial menu
+	var/static/list/ropes_designs
+
+	///Things this rope can transform into when it's tied to a person
+	var/obj/item/clothing/under/shibari/torso/shibari_body
+	var/obj/item/clothing/under/shibari/groin/shibari_groin
+	var/obj/item/clothing/under/shibari/full/shibari_fullbody
+	var/obj/item/clothing/shoes/shibari_legs/shibari_legs
+	var/obj/item/clothing/gloves/shibari_hands/shibari_hands
```

Comment:
Documentation needs to be applied to every var, not just one, be more descriptive.